arkouda/pdarrayclass.py

```diff
@@ -1307,10 +1308,15 @@ def _get_grouping_keys(self) -> List[pdarray]:
         API: this method must be defined by all groupable arrays, and it
         must return a list of arrays that can be (co)argsorted.
         '''
-        if self.dtype not in (akint64, akuint64):
-            raise TypeError("Grouping numeric data is only supported on integral types.")
+        if self.dtype == akbool:
+            from arkouda.numeric import cast as akcast
+            return [akcast(self, akint64)]
+        elif self.dtype in (akint64, akuint64):
+            return [self]
+        else:
+            raise TypeError("Grouping is only supported on numeric data (integral types) and bools.")
         # Integral pdarrays are their own grouping keys
```

Comment:
super small: this comment should be moved to be above `return [self]`

Reply:
Corrected comment location